### PR TITLE
Milestone: Redux setup

### PR DESCRIPTION
**Pull request by Kashif-Saleem-Ghuman**
In this milestone: 

- [ ] I have installed react-redux and redux toolkit
- [ ] Wrote book's actions and reducer using the [ducks pattern]
- [ ] - [ ] Wrote categories actions and reducer using the [ducks pattern]
- [ ] Defined action types for adding and removing a book.
- [ ] Set the initial state to be an empty array of books.
- [ ] Wrote reducer and exported it as default.
- [ ] Configured the Redux Store

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+// action types
+const ADDED_BOOK = 'book added';
+const DELETED_BOOK = 'book deleted';
+
```
Comment (m05e5) [inline src/redux/books/books.js]: - [ ] Hi. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 
 i.e  `const ADDED_BOOK = redux/books/ADDED_BOOK`(following your workflow)

File: src/redux/categories/categories.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Action types
+const CHECK_STATUS = 'check status';
```
Comment (m05e5) [inline src/redux/categories/categories.js]: - [ ] Hi. The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). 
 i.e  `const CHECK_STATUS = redux/books/CHECK_STATUS `(following your workflow)

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @Kashif-Saleem-Ghuman , 

Good job so far. !
There are some issues that you still need to work on to go to the next project but you are almost there!
mario_ladders


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please check the comments below 


_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@m05e5  Thank you for your valuable feedback. I will implement the change and submit again.

**Review by jerryowusu — APPROVED** [review]
Hi @Kashif-Saleem-Ghuman , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

CookedDelectableDolphin-size_restricted


## Highlights
All linters are passing :heavy_check_mark: 
Made the requested changes :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Kashif-Saleem-Ghuman** [comment]
@jerryowusu  Thank you for reviewing and approving my project. :)

**Pull request merged**